bcipy/tasks/session_data.py

```diff
@@ -19,6 +19,7 @@ class Inquiry:
         eeg_evidence - eeg evidence for each stimulus
         likelihood - combined likelihood for each stimulus
     """
+    EVIDENCE_SUFFIX = '_evidence'
```

Comment:
Maybe evidence names could live here too? We can use its name to get the values below.